keanu-examples/coalMiningDisasters/src/test/java/com/example/coal/ModelTest.java

```diff
@@ -16,7 +16,7 @@ public void shouldFindCorrectSwitchYear() {
         Model model = new Model(coalMiningDisasterData);
         model.run();
 
-        int switchYear = model.results.get(model.switchpoint).getMode();
+        int switchYear = model.results.getIntegerTensorSamples(model.switchpoint).getIntegerMode();
```

Comment:
Does 'getIntegerMode' add anything over 'getMode'? Seems very clear already that you're working with integers thanks to 'getIntegerTensorSamples'.